redaxo/src/core/lib/console/package/deactivate.php

```diff
@@ -15,7 +15,19 @@ protected function configure()
     {
         $this
             ->setDescription('Deactivates the selected package')
-            ->addArgument('package-id', InputArgument::REQUIRED, 'The id of the package (addon or plugin); e.g. "cronjob" or "structure/content"');
+            ->addArgument('package-id', InputArgument::REQUIRED, 'The id of the package (addon or plugin); e.g. "cronjob" or "structure/content"', null, static function() {
+                $packageNames = [];
+
+                foreach(rex_package::getRegisteredPackages() as $package) {
+                    if (!$package->isAvailable()) {
+                        continue;
+                    }
+
```

Comment:
Hier könnte man es abkürzen. Oder willst du es lieber einheitlich haben?

```suggestion
                foreach(rex_package::getAvailablePackages() as $package) {
```

Reply:
ich find es einheitlich gut, weil man dann auch die gegenstücke der jeweiligen commands gut sehen kann